.gitpod.yml

```
@@ -7,7 +7,7 @@ tasks:
       make sign-off
       pip install -e /workspace/kedro[test]
       cd /workspace
-      yes project | kedro new -s pandas-iris --checkout main
+      yes project | kedro new -s spaceflights-pandas --checkout main
```

Can we do

```suggestion
      kedro new --name project -s spaceflights-pandas --checkout main
```

already?

---

Ah yes I think that should work!